### ISSUE TITLE
Remove meta keywords from Prestashop - they are deprecated

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#11144]
In Traffic & SEO > SEO & URLs, we need to add an option in the SEO options block to allow the merchant to enable or disable metakeywords fields in every page where this field exists : 
- product page
- category page
- cms page
- brands page
- suppliers page

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

Initial issue : META keywords are deprecated

So this line of code from /templates/_partials/head.tpl file is useless and potentialy risky.

`<meta name="keywords" content="{block name='head_seo_keywords'}{$page.meta.keywords}{/block}">

![image](https://github.com/PrestaShop/PrestaShop/assets/43613217/e3d160e9-b99f-4f39-8a5d-83698ea4429b)

`
**Source**
https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html
https://datadrivenlabs.io/blog/meta-keyword-tag-google-2018/

**Reason?**
Not relevant keywords stuffing

**Comment by khouloudbelguith** [comment]
Hi @PrestaShark,

Thanks for your report.
@eternoendless, @PrestaShop/prestashop-core-developers what do you think?
Thanks!

**Comment by Rolige** [comment]
I think that this CAN’T be removed yet, the reason, all the SEO softwares (MOZ, SEMrush, etc.) still incorporating it in their ranks and reports.

**Comment by ghost** [comment]
Hi. Ok this is reason. I found article on SEMrush blog
https://www.semrush.com/blog/the-myths-behind-meta-keywords/

"In essence, nothing. Google ignores them, as does Yahoo. Bing looks at them, but not as a ranking signal. Bing reads the META Keywords tag as a spam signal. If your site uses the tag, but the keywords contained within it do not appear in your document, your site will be penalized."

"Using them potentially creates more harm than good. Close the book on using keyword meta tags." 

Second case. Keywords are removed from product SEO tab a long time ago too. What is main purpose to set keywords on other pages?

Shopify removed keywords from both product, main and other pages.
Like here:

view-source:https://debut-demo.myshopify.com/collections/featured-collection-on-home-page/products/key-pouch

view-source:https://debut-demo.myshopify.com/

Its just deprecated. Thats all :)

**Comment by Rolige** [comment]
I completely agree with you, meta keywords are no longer important to most of search engines, the question is, why are the most important SEO platforms still maintaining this in their analysis?, following this logic, it might be prudent to wait until they disappear completely, because I don’t think that using meta keywords now can affect in a site. That is just my view point!

**Comment by matks** [comment]
Thank you @Rolige and @PrestaShark for these important informations 😉 

Both of you have have brought very relevant items so this issue is tricky 😅

**Comment by BilalDr** [comment]
Hi everyone and thanks for all those informations. 

According to all those element, we all now that Google doesn't use the keywords in meta-keywords and we also know that websites using meta-keywords don't get penalized.

So it could be great to remove them but it's not the priority on SEO for the moment i think.
We will come back on this on a minor version.

**Comment by matthieu-rolland** [comment]
Hello @marionf, should we still do this ticket given BilalDr's last comment?

**Comment by marionf** [comment]
@breizoreol Maybe we should ask the opinion of SEO experts to be sure it's not usefull anymore and if it's we need to list all back-office pages where there are meta keywords fields.
@colinegin what do you think ?

**Comment by colinegin** [comment]
Thanks all for your feedbacks !
After a deeper analysis and a discussion with our SEO expert, we have decided to keep metakeywords as they might be useful in some cases. We are going to provide a new global SEO option to be able to enable or disable metakeywords, so that merchants who don't need it can disable it.

**Comment by centoasa** [comment]
why do I have the meta name="keywords" content = "bla blabla" in some products present and in others the field is empty? My release is 1.7.6.1
is there a way to edit and / or add keys to the meta keyword?

**Comment by SharakPL** [comment]
Are you kidding? 🤦‍♂️ If keywords no longer serve purpose for SEO then simply don't use it in meta tags. Removing it completely is **very bad idea**. I'd still want to set keywords for the product and load whole list of them on product page as a list of search links. How can I bring back keywords array on my shop?

> After a deeper analysis and a discussion with our SEO expert, we have decided to keep metakeywords as they might be useful in some cases.

@colinegin yes, they are useful. But where exactly do you keep it? I've searched everywhere. Found an option to enable them in SEO & URL, cleared cache, but they still don't show up. `$page.meta.keywords` array remains empty all the time, no matter how much keywords I'll add to my product.

BTW what's the purpose of this SEO & URL option for keywords. Is `<meta name="keywords" content="">` harmful in any way? IMO this option in BO is completely useless, especially when it's disabled by default. It should only affect meta tag visibility. `$page.meta.keywords` or better yet `$product.tags` or `$product.keywords` should be always generated.

**Comment by Hlavtox** [comment]
> After a deeper analysis and a discussion with our SEO expert, we have decided to keep metakeywords as they might be useful in some cases.

I think the SEO expert was 20 years in jail or coma. 😄😄 More info - https://yoast.com/meta-keywords/.

In 2000, they were already nearly dead. Meta keywords were supported only by two small insignificant search engines.
In 2009, Google released an official statement where they say they never cared about keywords at all - https://www.youtube.com/watch?v=jK7IPbnmvVU.
Then you have a Bing statement from 2014 - _“Today, it’s pretty clear the meta keyword tag is dead in terms of SEO value. Sure, it might have value for contextual ad systems or serve as a signal to bots plying the web looking for topics to target, but as far as search goes, that tag flat lined years ago as a booster.”_

**Comment by MatShir** [comment]
Let's remove it !!!!
